source/api_documentation/transactions_api/_rollup_message.rst

```diff
@@ -3,9 +3,9 @@
   <h2>
     Signal Transactions Rollup
   </h2>
-  <div class="alert alert-danger">
+  <div class="alert alert-warning">
     <p>
-      API versions <b>2020-10-01</b> and later only return rollup rows of transaction types "Call", "Sale", and "Post Call Event".
+      If you're upgrading from an API versions older than <b>2020-10-01</b> we suggest testing to familiarize yourself with the updated Signal data structure.
```

Comment:
minor typo: `from an API version older than` or `from API versions older than`

Reply:
Resolved the typo, thanks f
![Screen Shot 2022-09-19 at 8 56 56 AM](https://user-images.githubusercontent.com/18236509/191048077-d754f7de-83de-49f0-ae46-4e942a23947a.png)
or the extra 👀